packages/dx-react-scheduler/docs/guides/editing.md

```diff
@@ -0,0 +1,66 @@
+# React Scheduler - Editing
+
+The Scheduler supports editing features including creating, updating and deleting appointments. Use the corresponding plugins to manage the editing state. The editing state contains information about appointment currently being edited, changes applied to a particular appointment, and appointment that has been deleted and created but not yet committed. Once a user accepts an appointment addition, deletion, or changes made to an appointment (clicking the Save or Delete button), the Scheduler fires the `EditingState` plugin's `onCommitChanges` event and resets the appointment's editing state.
+
+## Related Plugins
+
+The following plugins implement editing features:
+
+- [EditingState](../reference/editing-state.md) - controls the editing state
+- [AppointmentTooltip](../reference/appointment-tooltip.md) - renders a appointment tooltip with editing controls
+- [AppointmentForm](../reference/appointment-form.md) - renders an appointment editing form
+- [Appointments](../reference/appointments.md) - renders appointments
```

Comment:
Should we point the `Appointments` plugin here? Usually, we mention only feature-specific plugins. For example [here](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/editing/) and [here](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/selection/). We don't refer the `Table` plugin everywhere.